File: app/src/main/java/com/sd/demo/tag_view/MainActivity.java
package com.sd.demo.tag_view;

import android.os.Bundle;

import androidx.appcompat.app.AppCompatActivity;

import com.sd.demo.tag_view.databinding.ActivityMainBinding;
import com.sd.lib.tag_view.FTagViewManager;

public class MainActivity extends AppCompatActivity
{
    private ActivityMainBinding mBinding;

    @Override
    protected void onCreate(Bundle savedInstanceState)
    {
        FTagViewManager.getDefault().setDebug(true);

        super.onCreate(savedInstanceState);
        mBinding = ActivityMainBinding.inflate(getLayoutInflater());
        setContentView(mBinding.getRoot());
    }

    @Override
    protected void onDestroy()
    {
        super.onDestroy();
        mBinding.viewTag.getItemManager().destroyItem();
    }
}

File: app/src/main/java/com/sd/demo/tag_view/MainActivity.kt
package com.sd.demo.tag_view

import android.os.Bundle
import android.view.View
import android.widget.Toast
import androidx.appcompat.app.AppCompatActivity
import com.sd.demo.tag_view.databinding.ActivityMainBinding
import com.sd.lib.tag_view.FTagViewManager

class MainActivity : AppCompatActivity(), View.OnClickListener {
    private val TAG = MainActivity::class.java.simpleName

    private lateinit var _binding: ActivityMainBinding

    override fun onCreate(savedInstanceState: Bundle?) {
        FTagViewManager.isDebug = true;
        super.onCreate(savedInstanceState)
        _binding = ActivityMainBinding.inflate(layoutInflater)
        setContentView(_binding.root)
    }

    override fun onClick(v: View?) {
        Toast.makeText(this, "click", Toast.LENGTH_SHORT).show()
    }

    override fun onDestroy() {
        super.onDestroy()
        _binding.tagView.destroyItem()
    }
}